UnrealPlugin/README.md

```diff
@@ -1,36 +1,112 @@
 This Unreal Plugin is implementing the GSDK directly in Unreal Engine.
 
-It was tested with Unreal Engine 4.26.2, but should work with other engines as well.
+It was tested with Unreal Engine 4.26.2, but should work with other engines as well. The sample game these instructions were created with was called ThirdPersonMP, so replace anywhere you see that with your game name.
+
+# Prerequisites:
+- Download Unreal Engine Source Build by following [these instructions](https://docs.unrealengine.com/4.26/en-US/ProgrammingAndScripting/ProgrammingWithCPP/DownloadingSourceCode/) from the Unreal website
+- Download Unreal Engine (this was tested on 4.26.2)
+- Download the [Unreal PlayfabGSDK Plugin folder](https://github.com/PlayFab/gsdk/pull/77/files)
+- Download Visual Studio
 
 # Setup
 
 ## Dependencies
 When using the plugin a few things need to be taken care off.
 
-First, when you are building dedicated server, make sure you add the following to DefaultGame.ini
+First, open your File Explorer and create a folder called “Plugins” in your games' root directory. Then drag the Playfab GSDK plugin file into the Plugins folder.
 
-```ini
-[/Script/UnrealEd.ProjectPackagingSettings]
-IncludeAppLocalPrerequisites=True
-```
+Right click on the .uproject file and choose the option to "Switch Unreal Engine version", which is how you can quickly check which Unreal Engine version you are currently using. 
+The popup seen below should appear. If you already see that the Unreal Engine version is source build, you don’t need to change anything, so click Cancel. If the Unreal version is not 
+currently the source build, select it from the dropdown list and then click OK. 
 
-If the category already exists in your DefaultGame.ini, then just add the second line to it. This ensures that all app local dependencies ship with the game as well.
+![image depicting a window that says "Select Unreal Engine Version"](Resources/UnrealTutorialImages/SelectUnrealEngineVersion.png)
+
+Right click on the .uproject file again and select “Generate Visual Studio Project Files".
 
-If you are using Continuous Integration (CI), then you could add it to your setup to only turn this flag on when building a dedicated server, so the additional dlls only get added if it is a dedicated server build.
+Then, double click to open the .uproject in Unreal.
 
-## Disabling plugins
-When you add Dedicated Server support to your project, you will definitely have created a \<projectname\>Server.Target.cs file.
+Using the Unreal development environment, go to Files->Create a new C++ class with whatever name you would like. In our example, we call this class MyGameInstance.
 
-In this file add the following lines to the constructor:
+Then close Unreal and generate project files in source build mode again.
+
+Then using Visual Studio, open those newly created files and add in the following methods for (MyGameInstance.cpp and MyGameInstance.h)
+
+#### MyGameInstance.cpp
 ```csharp
```

Comment:
this is C++ not C#

Reply:
Yes, meant to fix this but I wasn't sure if GitHub would recognize c++ or cplusplus so I just used csharp to format it! I think I can use c_cpp so will change that.